<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XlDOIDialogClass</class>
 <widget class="QDialog" name="XlDOIDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XlDOIDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="minLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="maxLineEdit"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGraphicsView" name="DOIHistogram">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
